Documentation/gettingstarted/microk8s.rst

```diff
@@ -0,0 +1,118 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    http://docs.cilium.io
+
+.. _gs_microk8s:
+
+******************************
+Getting Started Using MicroK8s
+******************************
+
+This guide uses `microk8s <https://microk8s.io/>`_ to demonstrate deployment
+and operation of Cilium in a single-node Kubernetes cluster. To run Cilium
+inside microk8s, a GNU/Linux distribution with kernel 4.9 or later is
+required (per the :ref:`admin_system_reqs`).
+
+Install microk8s
+================
+
+#. Install ``microk8s`` >= 1.14 as per microk8s documentation: `MicroK8s User
+   guide <https://microk8s.io/docs/>`_.
+
+#. Enable the microk8s DNS service
+
+   ::
+
+      microk8s.enable dns
+
+#. Configure microk8s to use CNI and allow Cilium to register as that CNI:
+
+   ::
+
+      echo "--allow-privileged" >> /var/snap/microk8s/current/args/kube-apiserver
+      sed -i 's/--network-plugin=kubenet/--network-plugin=cni/g'  /var/snap/microk8s/current/args/kubelet
+      sed -i 's/--cni-bin-dir=${SNAP}\/opt/--cni-bin-dir=\/opt/g'  /var/snap/microk8s/current/args/kubelet
+      sed -i 's/bin_dir = "${SNAP}\/opt/bin_dir = "\/opt/g'  /var/snap/microk8s/current/args/containerd-template.toml
+      rm /var/snap/microk8s/current/args/cni-network/cni.conf
+      systemctl restart snap.microk8s.daemon-containerd.service
+      systemctl restart snap.microk8s.daemon-apiserver.service
+      systemctl restart snap.microk8s.daemon-kubelet.service
+
+#. Install or configure ``kubectl``.
+
+   * Microk8s provides a version of kubectl, so if you don't otherwise have it
+     installed then you can simply alias the microk8s version:
+
+     ::
+
+        alias microk8s.kubectl kubectl
```

Comment:
Why not going with `snap alias microk8s.kubectl kubectl`?

Reply:
Done, thanks!